src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -29,14 +29,15 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
   handlers,
   tableClassName,
   tableStyle,
+  asc,
```

Comment:
Please add a default value to preserver the existing behavior:

```
asc = true,
```

Reply:
I think if you do this you can also remove your stuff again in the test mocks and stories? 
But please double check :-)

Reply:
Default value added.
I removed only from test mocks. I kept in stories otherwise is not visible in Controls tab below